Source/WebCore/css/makeSelectorPseudoClassAndCompatibilityElementMap.py

```diff
@@ -31,20 +31,14 @@
 def enumerablePseudoType(stringPseudoType):
     output = ['CSSSelector::PseudoClass::']
 
-    if stringPseudoType.endswith('('):
-        stringPseudoType = stringPseudoType[:-1]
-
-    internalPrefix = '-internal-'
-    if (stringPseudoType.startswith(internalPrefix)):
-        stringPseudoType = stringPseudoType[len(internalPrefix):]
-
+    # Use proper WebKit casing for enum.
     webkitPrefix = '-webkit-'
     if (stringPseudoType.startswith(webkitPrefix)):
-        stringPseudoType = stringPseudoType[len(webkitPrefix):]
+        stringPseudoType = stringPseudoType.replace('-webkit-', '-WebKit-')
```

Comment:
I’m not sure we should keep a local variable `webkitPrefix` if we only use it once.